packages/api/src/praise/entities.ts

```diff
@@ -31,6 +31,7 @@ const QuantificationModel = mongoose.model<QuantificationDocument>(
 const praiseSchema = new mongoose.Schema(
   {
     reason: { type: String, required: true },
+    reasonRealized: { type: String, required: false },
     sourceId: { type: String, required: true },
```

Comment:
I think `reasonRealized` should be required for all Praise entities. 

Then we can remove all related the conditionals all the way down through the frontend.

We'll need to write a migration to update all the praise in the database so they also have a `reasonRealized` value.